src/main.rs

```diff
@@ -75,6 +76,7 @@ pub fn handle_payload(inject_id: String, api: &Client, inject_data: &InjectRespo
                 &api,
                 inject_id.clone(),
                 &install_prerequisites,
+                &prerequisite.executor.clone(),
```

Comment:
Cloning a reference I'm not pretty sure, maybe you can make: prerequisite.executor.as_ref() ?